All_Relationships/utils/notebook_utils/bicluster.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+import re
+
```

Comment:
This my be a stylistic preference, just curious if there is a reason why you have divided the python functions into different files as opposed to having all the definitions in a single file with comments for the different functionalities.

Reply:
Main reason for this is that my mind works better when functions are in specified files compared to just a single universal file. Also, it helps with my organization of this repository.